Mathlib/Algebra/Order/Group/Basic.lean

```diff
@@ -34,6 +34,13 @@ lemma zpow_right_strictMono (ha : 1 < a) : StrictMono fun n : ℤ ↦ a ^ n := f
     _ < a ^ m * a ^ (n - m) := mul_lt_mul_left' (one_lt_zpow' ha <| Int.sub_pos_of_lt h) _
     _ = a ^ n := by simp [← zpow_add, m.add_comm]
 
+@[to_additive zsmul_strictAnti_left]
```

Comment:
```suggestion
@[to_additive zsmul_left_strictAnti]
```

Reply:
Oh, I guess this matches the name above. @YaelDillies, any thoughts on which name is right?

Reply:
Yes, I copied the name from above.

Reply:
Your name is better. The name above is vestigial